runtime/src/bank/address_lookup_table.rs

```diff
@@ -25,6 +26,20 @@ impl AddressLoader for &Bank {
     fn load_addresses(
         self,
         address_table_lookups: &[MessageAddressTableLookup],
+    ) -> Result<LoadedAddresses, AddressLoaderError> {
+        self.load_addresses_from_ref(
+            address_table_lookups
+                .iter()
+                .map(MessageAddressTableLookupRef::from),
+        )
+    }
+}
+
+impl Bank {
+    /// Load addresses from an iterator of `MessageAddressTableLookupRef`.
+    pub fn load_addresses_from_ref<'a>(
```

Comment:
Don't even need a trait - this is just a function on `Bank`. We will/can use this for resolving our transactions inside core/runtime.